controller/lib/core/sensors.h

```diff
@@ -43,14 +43,22 @@ class PressureSensors {
   // min/max possible reading from MPXV7002DP [kPa]
   inline constexpr static float DP_VAL_MIN = -2.0f;
   inline constexpr static float DP_VAL_MAX = 2.0f;
+
+  // Default diameters relating to Ethan's Alpha Venturi - II
+  // (https://docs.google.com/spreadsheets/d/1G9Kb-ImlluK8MOx-ce2rlHUBnTOtAFQvKjjs1bEhlpM/edit#gid=963553579)
+  // Port diameter must be larger than choke diameter [meters]
+  inline constexpr static const float DEFAULT_VENTURI_PORT_DIAM = 14e-3f;
+  inline constexpr static const float DEFAULT_VENTURI_CHOKE_DIAM = 5.5e-3f;
 };
```

Comment:
We can enforce the constraint that the port diameter is > the choke diam.  Could be here, or could be in sensors.cpp.

```
  static_assert(DEFAULT_VENTURI_PORT_DIAM > DEFAULT_VENTURI_CHOKE_DIAM)
```